src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -295,6 +301,8 @@ internal void InitWorker(RpcEvent startEvent)
 
         internal void WorkerReady(RpcEvent initEvent)
         {
+            _startLatencyMetric.Dispose();
```

Comment:
The lifetime of the `LanguageWorkerChannel` is much longer, so let's set this to `null` after disposing to avoid holding on to this reference